exporter/opentelemetry-exporter-otlp-proto-http/src/opentelemetry/exporter/otlp/proto/http/trace_exporter/encoder/__init__.py

```diff
@@ -0,0 +1,306 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from collections import abc
+from typing import Any, List, Optional, Sequence, Text
+
+from opentelemetry.proto.collector.trace.v1.trace_service_pb2 import (
+    ExportTraceServiceRequest as PB2ExportTraceServiceRequest,
+)
+from opentelemetry.proto.common.v1.common_pb2 import AnyValue as PB2AnyValue
+from opentelemetry.proto.common.v1.common_pb2 import (
+    ArrayValue as PB2ArrayValue,
+)
+from opentelemetry.proto.common.v1.common_pb2 import (
+    InstrumentationLibrary as PB2InstrumentationLibrary,
+)
+from opentelemetry.proto.common.v1.common_pb2 import KeyValue as PB2KeyValue
+from opentelemetry.proto.resource.v1.resource_pb2 import (
+    Resource as PB2Resource,
+)
+from opentelemetry.proto.trace.v1.trace_pb2 import (
+    InstrumentationLibrarySpans as PB2InstrumentationLibrarySpans,
+)
+from opentelemetry.proto.trace.v1.trace_pb2 import (
+    ResourceSpans as PB2ResourceSpans,
+)
+from opentelemetry.proto.trace.v1.trace_pb2 import Span as PB2SPan
+from opentelemetry.proto.trace.v1.trace_pb2 import Status as PB2Status
+from opentelemetry.sdk.trace import Event as SDKEvent
+from opentelemetry.sdk.trace import (
+    InstrumentationInfo as SDKInstrumentationInfo,
+)
+from opentelemetry.sdk.trace import Resource as SDKResource
+from opentelemetry.sdk.trace import Span as SDKSpan
+from opentelemetry.sdk.trace import SpanContext as SDKSpanContext
+from opentelemetry.trace import Link as SDKLink
+from opentelemetry.trace import SpanKind as SDKSpanKind
+from opentelemetry.trace import TraceState as SDKTraceState
+from opentelemetry.trace.status import Status as SDKStatus
+from opentelemetry.trace.status import StatusCode as SDKStatusCode
+from opentelemetry.util.types import Attributes as SDKAttributes
```

Comment:
I found the naming here confusing as these are all defined in the API and not SDKs

Reply:
Done in flow. I don't think we need `... import as` if there is no name collision. Will update this.